### PR TITLE
[php] tiny stylistic fixes in generator

### PR DESCRIPTION
**Pull request by vonagam**
Was looking at php generator source code, those things have caught my eye in top level functions.
Purely style things, no real changes.

### CODE REVIEW

File: src/generators/genphp7.ml [hunk 1]
```diff
@@ -1006,64 +991,42 @@ class abstract_wrapper (abstr) =
...
+		let wrapper = Hashtbl.find tbl key in
 		wrapper
 	with
 		| Not_found ->
-			let wrapper = new class_wrapper cls in
-			Hashtbl.add classes cls wrapper;
+			let wrapper = wrap key in
+			Hashtbl.add tbl key wrapper;
 			wrapper
 		| e -> raise e
```
Comment (RealyUniqueName) [inline src/generators/genphp7.ml]: This line can be removed too.

Comment (vonagam) [reply src/generators/genphp7.ml]: Done.

File: src/generators/genphp7.ml [hunk 2]
```diff
@@ -1293,7 +1241,7 @@ class code_writer (ctx:php_generator_context) hx_type_path php_name =
 			else if get_type_name type_path = "" then
 				match get_module_path type_path with
 				| [] -> "\\"
-				| _ -> "\\" ^ (String.concat "\\" (get_real_path (fst type_path))) ^ "\\"
+				| _ -> "\\" ^ (String.concat "\\" (get_real_path (get_module_path type_path))) ^ "\\"
 			else begin
 				let orig_type_path = type_path in
 				let type_path = match type_path with (pack, name) -> (pack, get_real_name name) in
```
Comment (RealyUniqueName) [inline src/generators/genphp7.ml:1247]: `get_module_path type_path` can be replaced with the matched value

**Review by RealyUniqueName — APPROVED** [review]

File: src/generators/genphp7.ml (hunk 2 above)
Comment (vonagam) [reply src/generators/genphp7.ml:1247]: Done.

**Pull request merged**